packages/shared/getComponentNameFromType.js

```diff
@@ -100,10 +100,10 @@ export default function getComponentNameFromType(type: mixed): string | null {
     switch (type.$$typeof) {
       case REACT_CONTEXT_TYPE:
         const context: ReactContext<any> = (type: any);
-        return getContextName(context) + '.Consumer';
-      case REACT_PROVIDER_TYPE:
-        const provider: ReactProviderType<any> = (type: any);
-        return getContextName(provider._context) + '.Provider';
+        return getContextName(context) + '.Provider';
```

Comment:
I imagine we'll be in a world soon where you never think about `.Provider` and just assume rendering a context provides values and "using" a context reads values. In this context does it make sense for the component name to carry forward this notion of `.Provider`? or should we switch it and call `FooContext.Provider` `FooContext` instead of calling `FooContext` `FooContext.Provider`?

Reply:
Hmm yea, I'd be happy to drop but figured it might invalidate snapshot tests or stuff like this. Maybe as a separate change later if this one settles?